modules/ossm-release-2-6-2.adoc

```diff
@@ -0,0 +1,42 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-6-2_{context}"]
+= {SMProductName} version 2.6.2
+
+This release of {SMProductName} includes 2.6.2, 2.5.5, and 2.4.11; addresses Common Vulnerabilities and Exposures (CVEs); contains bug fixes; and is supported on {product-title} 4.14 and later.
+
+include::snippets/ossm-current-version-support-snippet.adoc[]
+
+The most current version of the {KialiProduct} can be used with all supported versions of {SMProductName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource.  The version of {SMProductShortName} automatically ensures a compatible version of Kiali.
+
+[id=ossm-release-2-6-2-components_{context}]
+== Component updates
+
+|===
+|Component |Version
+
+|Istio
+|1.20.8
+
+|Envoy Proxy
+|1.28.5
+
+|Kiali Server
+|1.73.14
+|===
+
+[id="ossm-new-features-2-6-2_{context}"]
+== New features
+
+* https://issues.redhat.com/browse/OSSM-8032[OSSM-8032] The cert-manager tool is now supported on IBM Power®, IBM Z®, and IBM® LinuxONE.
```

Comment:
Rephrase cert-manager as:

This release supports the cert-manager tool on {ibm-power-title}, {ibm-z-title}, and {ibm-linuxone-name}.